src/controllers/tasks.ts

```diff
@@ -0,0 +1,20 @@
+import { Response } from 'express';
+import { CustomError } from '../helper';
+import { getTasksByUserId } from '../database/query/tasks';
+import { TokenRequest } from '../interfaces';
+
+const getTasks = (req: TokenRequest, res: Response): void => {
+  const userId = req.userData?.id;
+  getTasksByUserId(+userId!)
+    .then((tasks) => {
+      res.status(200).json({
+        message: 'Tasks retrieved successfully',
+        data: tasks,
```

Comment:
we need to return the rows array here, not all the query result

```suggestion
        data: tasks.rows,
```